### PR TITLE
Session 1 to 5 tasks

### PR DESCRIPTION
**Pull request by dheeraj-anumula**
Completed javascript coding tasks till session 2.

### CODE REVIEW

File: Session 1/task3.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+http://www.codewars.com/kata/basic-mathematical-operations
+*/
+
+function basicOp(operation, value1, value2)
+{
+  // Code
+  var result;
+  switch(operation){
+    case '+':
+      result=value1+value2;
```
Comment (rhushi-lokhande) [inline Session 1/task3.js]: use return instead of result

File: Session 3/task1.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/*
+http://www.codewars.com/kata/prefill-an-array
+*/
+
+function prefill(n, v) {
+  
+  if(n != parseInt(n, 10) || n<0 ){
```
Comment (rhushi-lokhande) [inline Session 3/task1.js:7]: user isNaN function

File: Session 3/task3.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*
+http://www.codewars.com/kata/closures-and-scopes/train/javascript
+*/
+function createFunctions(n) {
+  var callbacks = [];
+
+  for (var i=0; i<n; i++) {
+
```
Comment (rhushi-lokhande) [inline Session 3/task3.js:8]: use let and arrow function

Comment (dheeraj-anumula) [reply Session 3/task3.js:8]: We did not cover ES6 topics when I completed this task. Completed now.

File: Session 1/task3.js (hunk 1 above)
Comment (dheeraj-anumula) [reply Session 1/task3.js]: Made changes

File: Session 3/task1.js (hunk 2 above)
Comment (dheeraj-anumula) [reply Session 3/task1.js:7]: Here in this problem, when n is "0" or when n is "1" or something like that it should be converted to integer to pass the test cases. So we have to parse it to integer to make it work.

**Review by dheeraj-anumula — COMMENTED** [review]
Made related changes for the given review

**Pull request merged**